vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,19 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting
+	Output     string // Output from last check
```

Comment:
Actually, this has to be `[]byte`, so it gets armored against bad or binary output.